jupyter-extension/src/plugins/mount/components/Pipeline/hooks/usePipeline.tsx

```diff
@@ -47,7 +47,7 @@ export const usePipeline = (
   const [pipelineProject, setPipelineProject] = useState('');
   const [imageName, setImageName] = useState('');
   const [inputSpec, setInputSpec] = useState('');
-  const [pipelinePort, setPipelinePort] = useState('');
+  const [pipelinePort, setPipelinePort] = useState(-1);
```

Comment:
I think whether the not the port should be a string is up to interpretation, but I don't think the default state should be invalid.